src/Manager/ManagerBase.php

```diff
@@ -40,4 +54,28 @@ protected function getResponseJson(RequestInterface $request)
         return json_decode($body, true);
     }
 
+    /**
+     * Get query string of using the options.
+     *
+     * @param $options The options
+     * @return string  The query string
+     */
+    protected function getQueryString($options) {
+      $queries = [];
+      foreach ($this->queryParameters as $queryName => $queryDefaultValue) {
+        // Use user value if possible.
+        if (isset($options[$queryName])) {
+          $queries[] = $queryName . '=' . $options[$queryName];
+          continue;
+        }
+        // Use default value if possible.
+        if (!isset($options[$queryName]) && is_string($queryDefaultValue)) {
```

Comment:
Doesn't seem like you need the `!isset($options[$queryName])` check as it is checked above and followed by a `continue`.